src/controller.rs

```diff
@@ -153,6 +159,18 @@ impl Controller for FoxBox {
         format!("ws://{}:{}/services/{}/", self.hostname, self.ws_port, service_id)
     }
 
+    fn get_config(&self, namespace: String, property: String, default: String)
+        -> String {
```

Comment:
So let's go with .get_config_with_default() and .get_config().  But get_config_with_default() is basically .get_config().unwrap_or(default) which is not so ugly in my view.